doc/source/whatsnew/v0.20.0.txt

```diff
@@ -50,6 +50,7 @@ Other enhancements
 - ``Series.sort_index`` accepts parameters ``kind`` and ``na_position`` (:issue:`13589`, :issue:`14444`)
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
```

Comment:
can you add a mini section to the docs and put a pointer here (or is the doc-string enough)?